app/services/github_service.rb

```diff
@@ -0,0 +1,23 @@
+class GithubService
+
+  def initialize(user)
+    @_conn = Faraday.new("https://api.github.com/") do |faraday|
+      faraday.params['Authorization'] = "token #{user.token}"
+    end
+  end
+
+  def starred_repos(user)
+    response = conn.get("/users/#{user.username}/starred")
+    parse(response.body)
+  end
```

Comment:
having issues with this method...response is nil. when I pry into the response, the token I see in the request doesn't match the token I'm using in postman.